ranger-metrics/src/main/java/org/apache/ranger/metrics/sink/RangerMetricsJsonSink.java

```diff
@@ -66,8 +66,8 @@ record = new HashMap<> ();
                 }
             } else {
                 if (LOG.isDebugEnabled()) {
-		    LOG.debug("=== RangerMetricsJsonSink:putMetrics(): skipping... "+ metricsRecord.context());
-		}
+                    LOG.debug("=== RangerMetricsJsonSink:putMetrics(): skipping... " + metricsRecord.context());
```

Comment:
Replace string concat with parameterized log message:
```
LOG.debug("=== RangerMetricsJsonSink:putMetrics(): skipping... {}", metricsRecord.context());
```

 With this, the surrounding `if (LOG.isDebugEnabled()) {` can be removed as well. Please review and update other occurrences as well.